docs/README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Overview
 
-- For background, including which branch to work on, see ["Updating documentation" in CONTRIBUTING](../CONTRIBUTING.md#updating-documentation)
-- The documentation is written in Markdown
-- It is converted to HTML via Ruby and Jekyll
-  - Important: Ruby 2 is required, for example, Ruby 2.7
-- The published documentation is at <https://schemar.github.io/obsidian-tasks/>
+-   For background, including which branch to work on, see ["Updating documentation" in CONTRIBUTING](../CONTRIBUTING.md#updating-documentation)
```

Comment:
I wondered where these extra spaces were coming from.
There's a curious cycle going on that your edits include extra spaces in bullet lists, and then @sytone (and I think mine) remove them.

It's is only cosmetic, but it does make git annotate/blame rather less useful, and makes reviews take longer, as you have to mentally separate out formatting changes from content ones...

Reply:
I think it's my LSP based md formatter 🤔